blocks/monitoring/evaluators.py

```diff
@@ -50,8 +76,6 @@ def __init__(self, variables, use_take_last=False):
         self.use_take_last = use_take_last
 
         self.variable_names = [v.name for v in self.variables]
-        if len(self.variable_names) < len(self.variables):
```

Comment:
Leave this check as well: sometimes `AggergationBuffer` is used on its own, e.g. in `TrainingDataMonitoring`.